### ISSUE TITLE
Visual Studio Code "UserSetup" breaks opening of Visual Studio projects and solution files (in Administrator mode)

### ISSUE THREAD

**Issue by nenadvicentic** [madskristensen/OpenInVsCode#21]
Hi there,

after I was prompted by Installation Wizzard to reinstall Visual Studio Code (v1.26.1), as new recommended "UserSetup", next time I run "Open In Visual Studio Code" on `*.csproj` file, it prompted me to provide new  `code.exe` location. Now that it is setup to:

>C:\users\\{username\}\AppData\Local\Programs\Microsoft VS Code\Code.exe

Now, when I righ-click on `*.sln` or `*.csproj` files, Visual Studio Code opens folder:

>C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\Common7\IDE\

This happens only when I use **Visual Studio in Administrator mode** (which is almost 100% of time).

When I open folders inside of the project (ex: `Controllers`) path is correct. Option `Open solution/project as regular file` is set to `False` and `Command line arguments` are empty.

**Comment by mrchops1024** [comment]
The same thing is happening to me, except I'm not launching Visual Studio as administrator AND I'm using the same global install of VSCode (not the new user one). I can open individual files in VSCode but if I try to open a project or solution it opens:

>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE

The only thing that's changed recently is that VSCode was updated to 1.26.1. I tried rolling back to 1.25.1 and the issue persists.
